internal/controller/volumereplicationgroup_controller.go

```diff
@@ -668,10 +680,18 @@ func (v *VRGInstance) updatePVCList() error {
 		return nil
 	}
 
-	if err := v.updateReplicationClassList(); err != nil {
-		v.log.Error(err, "Failed to get VolumeReplicationClass list")
+	if rmnutil.IsCGEnabled(v.instance.GetAnnotations()) {
```

Comment:
@BenamarMk if the CG annotation is added to an already created and reconciled DRPC/VRG, will it cause issues here. I.e now the VRG reconciliation would shift to CG based processing and potentially leave stale non-CG resources (at least in the Volrep case).

IOW, CG enabling is during initial reconcile from DRPC and should not be enabled later, is that somehow prevented overall from DRPC creating the VRG?

Reply:
yes, I'm waiting for this pr merged, then I can rebase my pr to adopt this func.